sessionctx/variable/varsutil.go

```diff
@@ -171,19 +202,10 @@ func ValidateSetSystemVar(vars *SessionVars, name string, value string) (string,
 		return value, UnknownSystemVar.GenByArgs(name)
 	}
 	switch name {
+	case ConnectTimeout:
```

Comment:
So many magic numbers in this function. Maybe we can define some constants for it.

Reply:
Most of the number literals are system variable specific and can hardly be reused, I think make them constants help little.